build_tools/docker/bazel-tensorflow-swiftshader/Dockerfile

```diff
@@ -16,8 +16,14 @@
 # running Vulkan tests on SwiftShader.
 
 FROM gcr.io/iree-oss/bazel-tensorflow-vulkan AS final
-
 COPY --from=gcr.io/iree-oss/swiftshader swiftshader/ swiftshader/
 
 # Set VK_ICD_FILENAMES so Vulkan loader can find the SwiftShader ICD.
 ENV VK_ICD_FILENAMES /swiftshader/vk_swiftshader_icd.json
+
+COPY --from=gcr.io/iree-oss/vulkan /opt/vulkan-sdk/ /opt/vulkan-sdk/
+ARG VULKAN_SDK_VERSION=1.2.154.0
+ENV VULKAN_SDK="/opt/vulkan-sdk/${VULKAN_SDK_VERSION}/x86_64"
+ENV PATH="${VULKAN_SDK}/bin:$PATH"
+RUN ln -s "${VULKAN_SDK}/lib/libvulkan.so" /usr/lib/x86_64-linux-gnu/ \
+  && ln -s "${VULKAN_SDK}/lib/libvulkan.so.1" /usr/lib/x86_64-linux-gnu/
```

Comment:
This is needed for check_vulkan.sh, which calls `vulkaninfo`. And we want to use the binaries in the same vulkan SDK?

Reply:
This is already part of the image because it inherits from https://github.com/google/iree/blob/main/build_tools/docker/bazel-tensorflow-vulkan/Dockerfile

Reply:
It actually don't work if I revert the change. I can see `vulkaninfo` in `/opt/vulkan-sdk/1.2.154.0/x86_64/bin`, but the ENV is not set. It does fail on CI.

I thought the ENV would be inherit but seems not, any ideas?

Reply:
Hmmmm let me investigate

Reply:
Ok I have no idea why this is failing, but I got fed up debugging docker images and pinned everything to digests (https://github.com/google/iree/pull/4047). After that, I was able to trivially update the swiftshader version (https://github.com/google/iree/pull/4059). Sorry for all the churn on this PR, but I think it can now be closed in favor of that one.